### ISSUE TITLE
Is there anyway I can use style with scoped in scss? 

### ISSUE THREAD

**Issue by TimothyPrachasri** [vuesion/vuesion#422]
Is there anyway i can use style with scoped in scss?. since I've found that the rules for sass loader allows only css module to be used. However, is there anyway I can at least overwrite some webpack rules of scss from using css module to using any style (scoped or global).

**Comment by devCrossNet** [comment]
Hey @TimothyPrachasri,
you can modify the webpack config based on https://github.com/vuesion/packages/blob/master/packages/service/src/webpack/config/base.ts in `./vuesion/webpack.config.js`.

In your case you have to replace the contents of the custom webpack config with:

```
/**
 * modify webpack config
 * @param {webpack.Configuration} config - webpack config
 * @param {string} target - build target, can  be "client" or "server"
 * @returns {*}
 */
const isProd = process.env.NODE_ENV === 'production';

module.exports = (config, target) => {
  if (target === 'client' && isProd) {
    config.performance = {
      hints: 'warning',
      maxEntrypointSize: 400000,
      maxAssetSize: 300000,
    };
  }

  config.module.rules[2] = {
    test: /\.scss$/,
    oneOf: [
      {
        resourceQuery: /module/,
        use: [
          'vue-style-loader',
          {
            loader: 'css-loader',
            options: {
              modules: true,
              localIdentName: '[local]_[hash:base64:5]',
            },
          },
          {
            loader: 'postcss-loader',
            options: {
              ident: 'postcss',
              plugins: () => [
                require('autoprefixer')({ browsers: ['last 2 versions', 'ie >= 11'] }),
                require('css-mqpacker')(),
                require('cssnano')({
                  preset: [
                    'default',
                    {
                      discardComments: {
                        removeAll: true,
                      },
                      zindex: false,
                      normalizeWhitespace: isProd,
                    },
                  ],
                }),
              ],
            },
          },
          'sass-loader',
        ],
      },
      {
        use: [
          'vue-style-loader',
          'css-loader',
          {
            loader: 'postcss-loader',
            options: {
              ident: 'postcss',
              plugins: () => [
                require('autoprefixer')({ browsers: ['last 2 versions', 'ie >= 11'] }),
                require('css-mqpacker')(),
                require('cssnano')({
                  preset: [
                    'default',
                    {
                      discardComments: {
                        removeAll: true,
                      },
                      zindex: false,
                      normalizeWhitespace: isProd,
                    },
                  ],
                }),
              ],
            },
          },
          'sass-loader',
        ],
      },
    ],
  };

  return config;
};
```

Let me know if this helps you.

**Comment by devCrossNet** [comment]
The solution posted enables both css-modules and global/scoped css, in case you want to use existing components that rely on css-modules.

**Comment by devCrossNet** [comment]
@TimothyPrachasri did it work?

**Comment by TimothyPrachasri** [comment]
Right now i’m trying to use css module as a workaround. If i change to
scoped scss, will tell you  asap.

**Comment by devCrossNet** [comment]
closed, this feature is now included in @vuesion/service version `3.0.0-alpha.6`
